themes/mongodb/src/main/controller.js

```diff
@@ -3,6 +3,7 @@ import * as componentAtlas from '../js/componentAtlas';
 import * as componentCloseOpen from '../js/componentCloseOpen';
 import * as componentCodeBlockFix from '../js/componentCodeBlockFix';
 import * as componentCopyButtons from '../js/componentCopyButtons';
+import * as componentEcosystemLinkTracK from '../js/componentEcosystemLinkTracK';
```

Comment:
`K` should be lowercase. This _happens_ to work on macOS because HFS is case-insensitive by default, but we should be able to build this on any Unix.

Reply:
wow good catch my bad